### PR TITLE
[Trivial] Fix svace and coverity issue @open sesame 02/21 14:35

### PR DESCRIPTION
**Pull request by baek2sm**
- add swap_lookahead to default constuctor of Manager
- add cache_swap to default constructor of ProfileEventData

**Self evaluation:**
1. Build test: [x]Passed [ ]Failed [ ]Skipped
2. Run test: [x]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Review by DonghakPark — APPROVED** [review]
LGTM!

File: nntrainer/utils/profiler.h [hunk 1]
```diff
@@ -103,10 +103,11 @@ struct ProfileEventData {
    *
    */
   ProfileEventData(int item, size_t cur, size_t total, std::string str,
-                   std::chrono::microseconds dur) :
+                   std::chrono::microseconds dur, bool swap = false) :
     time_item(item),
     alloc_current(cur),
     alloc_total(total),
+    cache_swap(swap),
```
Comment (jihochu) [inline nntrainer/utils/profiler.h:110]:
```suggestion
    cache_swap(false),
```

File: nntrainer/tensor/manager.h [hunk 2]
```diff
@@ -130,7 +130,9 @@ class Manager {
   /**
    * @brief     Constructor of Manager
    */
-  Manager() : enable_optimizations(true) {}
+  Manager(unsigned int lookahead = 0) :
+    enable_optimizations(true),
+    swap_lookahead(lookahead) {}
```
Comment (jihochu) [inline nntrainer/tensor/manager.h:135]:
```suggestion
    swap_lookahead(0) {}
```

**Comment by SeoHyungjun** [comment]
LGTM

**Review by lhs8928 — APPROVED** [review]
LGTM

File: nntrainer/utils/profiler.h (hunk 1 above)
Comment (baek2sm) [reply nntrainer/utils/profiler.h]: I applied your suggestion. thanks.

File: nntrainer/tensor/manager.h (hunk 2 above)
Comment (baek2sm) [reply nntrainer/tensor/manager.h]: I applied your suggestion. thank you.

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**